.ci/cico_do_docker_build_tag_push.sh

```diff
@@ -15,6 +15,10 @@ RH_CHE_TAG=$(git rev-parse --short HEAD)
 UPSTREAM_TAG=$(sed -n 's/^SCM-Revision: \(.\{7\}\).*/\1/p' ${ABSOLUTE_PATH}/../assembly/assembly-wsmaster-war/target/war/work/org.eclipse.che/assembly-wsmaster-war/META-INF/MANIFEST.MF)
 
 # Now lets build the local docker images
+
+ADDONS=${ABSOLUTE_PATH}/../openshift/minishift-addons
+KEYCLOAK_CONFIGURATOR_IMAGE="rh-che-standalone-keycloak-configurator"
```

Comment:
maybe `KEYCLOAK_STANDALONE_CONFIGURATOR_IMAGE` is a better name ?

Reply:
yes, you're right

Reply:
done